README.md

```diff
@@ -1,18 +1,40 @@
-# DRAFT ONLY : UNTIL KEP IS MERGED 
-  https://github.com/kubernetes/enhancements/pull/1383
+COSI repos:
+
+|Repo | Images | 
+|----------------|-----------|
+| [Spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec) | N/A  |
+| [API](https://github.com/kubernetes-sigs/container-object-storage-interface-api) | N/A |
+| [Controller](https://github.com/kubernetes-sigs/container-object-storage-interface-controller) | [cosi-controller](https://quay.io/repository/containerobjectstorage/objectstorage-controller?tab=tags) |
+| [Provisioner Sidecar](https://github.com/kubernetes-sigs/container-object-storage-interface-provisioner-sidecar) | cosi-provisioner TODO |
+| [CSI Adapter](https://github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter) | cosi-node-adapter TODO |
+\
+COSI deprecated repos:
+
+[Spec](https://github.com/container-object-storage-interface/spec) \
+[API](https://github.com/container-object-storage-interface/api) \
+[Manager (i.e. Controller)](https://github.com/container-object-storage-interface/cosi-controller-manager) \
+[Provisioner Sidecar](https://github.com/container-object-storage-interface/cosi-provisioner-sidecar) \
+[Ephemeral CSI Driver](https://github.com/container-object-storage-interface/ephemeral-csi-driver) 
+
```

Comment:
The table does not look right:

https://github.com/kubernetes-sigs/container-object-storage-interface-spec/blob/63d3ccc27e80c3eb2db3b09520d7665716cff5f7/README.md